docs/en/operations/settings/settings.md

```diff
@@ -769,6 +769,39 @@ Example:
 log_query_threads=1
 ```
 
+## log_comment {#settings-log-comment}
+
+Logs the comment into [system.query_log](../system-tables/query_log.md) table and server log.
```

Comment:
Specifies the value for the `log_comment` field of the [system.query_log](../system-tables/query_log.md) table and comment text for the server log.